install/kubernetes/cilium/charts/hubble-ui/templates/svc.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.global.hubble.ui.enabled }}
+kind: Service
+apiVersion: v1
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: hubble-ui
+spec:
+  selector:
+    k8s-app: hubble-ui
+  ports:
+    - name: http
+      port: 12000
```

Comment:
I think we want port 80/443 for hubble-ui but this can be addressed in a follow-up PR (cc @geakstr @michi-covalent)